chain/client/src/sync.rs

```diff
@@ -770,6 +781,37 @@ impl StateSync {
                         }
                     }
                 }
+                ShardSyncStatus::StateDownloadApplying => {
```

Comment:
Following the same logic, this stage can be called `StateDownloadFinalizeState`.

Or, I think it can just be merged to the next stage 'StateDownloadComplete`. Both seem fine to me.

Reply:
While we are in this state while we apply them in other thread. So I think my naming is more in line with convention